android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java

```diff
@@ -181,7 +181,7 @@ private void setProps(@NonNull Bundle newProps) {
      * by/associated with the specified {@code eventName}.
      */
     private void maybeSetViewURL(String eventName, ReadableMap eventData) {
-        String url = eventData.getString("url");
+        String url = eventData.hasKey("url") ? eventData.getString("url") : null;
```

Comment:
now Im thinking what difference does this code make ? what would be returned by eventData if there is no url ?

Reply:
It throws an exception.

Reply:
weird